docs/src/configuration/services/vault.md

```diff
@@ -14,88 +14,23 @@ The Vault key management service (KMS) provides key management and access contro
 
 ## Add Vault
 
-### Add the service
-
-In your `.platform/services.yaml` file, add the service:
-
-```yaml
-vault-kms:
-    type: vault-kms:1.6
-    disk: 2048
-
-    configuration:
-        endpoints:
-            <ENDPOINT_ID>:
-                - policy: admin
-                  key: "<KEY_NAME>"
-                  type: sign
-                - policy: sign
-                  key: "<KEY_NAME>"
-                  type: sign
-                - policy: verify
-                  key: "<KEY_NAME>"
-                  type: sign
-```
-
-What this includes:
+{{% endpoint-description type="vault-kms" noApp=true %}}
```

Comment:
I can use something other than the `type` (and `policy`) for sure. I wouldn't use `demo-app` because it's basically like using lorem ipsum text and I'd prefer meaningful, but avoiding confusing is good :+1:

Reply:
Also, the current app configuration is invalid. Will be fixed in version 1.3 of the parser. 

And I'll go back and fix `demo-app` above.

Reply:
I think that following back each example like on the other pages helps to understand the logic. Thanks for the change :)